packages/core/src/lib/actions/copy.action.ts

```diff
@@ -0,0 +1,21 @@
+import { DiagramEngineCore } from '../engine.core';
+import { LinkModel, NodeModel, PointModel, PortModel } from '../models';
+import { BaseAction } from './base.action';
+
+export class CopyAction extends BaseAction {
```

Comment:
this is exactly the same as the selected action, why not invoke the same selection action and on the copy just get the information from the previous selection action? instead of duplicating the code?